controllers/datadogagent/feature/remoteconfig/feature.go

```diff
@@ -53,15 +56,21 @@ func (f *rcFeature) ID() feature.IDType {
 func (f *rcFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.RequiredComponents) {
 	f.owner = dda
 
-	if dda.Spec.Features != nil && dda.Spec.Features.RemoteConfiguration != nil && apiutils.BoolValue(dda.Spec.Features.RemoteConfiguration.Enabled) {
-		f.enabled = true
-		reqComp = feature.RequiredComponents{
-			Agent: feature.RequiredComponent{
-				IsRequired: apiutils.NewBoolPointer(true),
-				Containers: []apicommonv1.AgentContainerName{
-					apicommonv1.CoreAgentContainerName,
-				},
+	reqComp = feature.RequiredComponents{
+		Agent: feature.RequiredComponent{
+			IsRequired: apiutils.NewBoolPointer(true),
+			Containers: []apicommonv1.AgentContainerName{
+				apicommonv1.CoreAgentContainerName,
 			},
+		},
+	}
+
+	if dda.Spec.Features != nil && dda.Spec.Features.RemoteConfiguration != nil && dda.Spec.Features.RemoteConfiguration.Enabled != nil {
+		// If a value exists, explicitely enable or disable Remote Config and override the default
+		if apiutils.BoolValue(dda.Spec.Features.RemoteConfiguration.Enabled) {
+			f.enabled = true
+		} else {
+			f.enabled = false
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
		f.enabled = apiutils.BoolValue(dda.Spec.Features.RemoteConfiguration.Enabled)
```